.github/workflows/build.yml

```
@@ -98,3 +98,38 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    name: Run Go Package Tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Download Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '^1.20.2'
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - name: "Retrieve package location"
+        run: |
+            echo PACKAGE_LOCATION="$(find ./ -name 'test')/../" >> $GITHUB_OUTPUT
+        id: packagelocation
+
+      - name: Generate Mocks and Run Tests
```

Standardize step name capitalization:
```suggestion
      - name: Generate mocks and run tests
```